.github/workflows/e2e.yml

```
@@ -53,3 +53,7 @@ jobs:
           DATABASE_URL: "postgres://postgres@localhost:5432/postgres"
         working-directory: ./e2e/packages/sync-test
         run: pnpm test:ci
+
+      - name: Run contracts tests
+        working-directory: ./e2e/packages/contracts
+        run: pnpm test:ci
```

```suggestion
      - name: Run tests
        working-directory: ./e2e
        run: pnpm run test:ci
```
what if we just run em all?